templates/lb-fargate-service/manifest.yml

```diff
@@ -1,51 +1,48 @@
-# Your application name will be used in naming your resources
-# like log groups, services, etc.
+# The manifest for the "{{.Name}}" application.
```

Comment:
@clareliguori Do you think these configurations are sufficient for a "frontend" pattern? Is there anything you'd want to see/change that'd be a common usecase?